src/main/java/Chris.java

```diff
@@ -0,0 +1,116 @@
+import java.util.Scanner;
+
+import task.Deadlines;
+import task.Events;
+import task.TaskLists;
+import task.Tasks;
+import task.ToDos;
+import exceptions.InputException;
+public class Chris {
+    public static void main(String[] args) throws InputException {
+        String logo = "      ___           ___           ___                       ___     \n" +
+                "     /\\  \\         /\\__\\         /\\  \\          ___        /\\  \\    \n" +
+                "    /::\\  \\       /:/  /        /::\\  \\        /\\  \\      /::\\  \\   \n" +
+                "   /:/\\:\\  \\     /:/__/        /:/\\:\\  \\       \\:\\  \\    /:/\\ \\  \\  \n" +
+                "  /:/  \\:\\  \\   /::\\  \\ ___   /::\\~\\:\\  \\      /::\\__\\  _\\:\\~\\ \\  \\ \n" +
+                " /:/__/ \\:\\__\\ /:/\\:\\  /\\__\\ /:/\\:\\ \\:\\__\\  __/:/\\/__/ /\\ \\:\\ \\ \\__\\\n" +
+                " \\:\\  \\  \\/__/ \\/__\\:\\/:/  / \\/_|::\\/:/  / /\\/:/  /    \\:\\ \\:\\ \\/__/\n" +
+                "  \\:\\  \\            \\::/  /     |:|::/  /  \\::/__/      \\:\\ \\:\\__\\  \n" +
+                "   \\:\\  \\           /:/  /      |:|\\/__/    \\:\\__\\       \\:\\/:/  /  \n" +
+                "    \\:\\__\\         /:/  /       |:|  |       \\/__/        \\::/  /   \n" +
+                "     \\/__/         \\/__/         \\|__|                     \\/__/    ";
+        System.out.println("Hello from \n" + logo );
+
+        Scanner sc = new Scanner(System.in);
+        String command = "";
+        TaskLists listCommands = new TaskLists();
+        System.out.println("____________________________________________________________\n" +
+                "Hello, I'm Chris\n" +
+                "What can I do for you?\n" +
+                "____________________________________________________________\n");
+        while (sc.hasNextLine()) {
+            try {
+                command = sc.nextLine();
+                if (command.equals("bye")) {
+                    System.out.println("____________________________________________________________\n" +
+                            "Bye. Hope to see you again soon!\n" +
+                            "____________________________________________________________");
+                    break;
+                } else if (command.equals("list")) {
+                    System.out.println("____________________________________________________________");
```

Comment:
Repetition of the code is not good. Perhaps, you can consider storing this horizontal line in some variable and printing that variable when you need to print this line.